js/tests/components/__snapshots__/UpsellBox.test.js.snap

```diff
@@ -52,40 +52,67 @@ exports[`UpsellBox renders the snippet editor inside of it 1`] = `
       </a>
        test
     </p>
-    <UpsellBox__StyledList
+    <.UpsellBox__StyledList-gjf66g-0
```

Comment:
Why is there a `.` in front of this component?

Reply:
No one knows.